bycycle/plts/burst.py

```diff
@@ -0,0 +1,227 @@
+"""Plot burst detection parameters."""
+
+from itertools import cycle
+
+import numpy as np
+from scipy.stats import zscore
+
+import matplotlib.pyplot as plt
+
+from neurodsp.plts import plot_time_series, plot_bursts
+
+from bycycle.plts.cyclepoints import plot_cyclepoints
+from bycycle.plts.utils import apply_tlims, get_extrema
+
+###################################################################################################
+###################################################################################################
+
+def plot_burst_detect_summary(df, sig, fs, osc_kwargs, tlims=None, figsize=(15, 3),
+                              plot_only_result=False, burst_params=None, interp=True):
+    """
+    Create a plot to study how the cycle-by-cycle burst detection
+    algorithm determines bursting periods of a signal.
+
+    Parameters
+    ----------
+    df : pandas.DataFrame
+        Dataframe output of :func:`~.compute_features`.
+    sig : 1d array
+        Time series to plot.
+    fs : float
+        Sampling rate, in Hz.
+    osc_kwargs : dict
```

Comment:
I'm not so sure about the name of this inputs (and, relatedly, the `burst_params` input below). We don't need to maintain the same labels from before (this is a breaking update anyways). 

If this is the same as `burst_detection_kwargs`, I think it should be named the same. And `burst_params` seems like it's custom names or something, so I think there would be a better name there. Maybe just `burst_param_labels`?